rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/message_introspection.hpp

```diff
@@ -56,17 +80,30 @@ typedef struct ROSIDL_TYPESUPPORT_INTROSPECTION_CPP_PUBLIC MessageMember_s
   ///
   /// Available for array and sequence members.
   void (* assign_function)(void *, size_t index, const void *);
+  /// If is_array_ is true, a pointer to a function that resizes the array.
+  /// First argument should be a pointer to the actual memory representation of the member.
+  /// Only used for nested interface types.
   void (* resize_function)(void *, size_t size);
 } MessageMember;
 
+/// Structure used to describe all fields of a single interface type.
 typedef struct ROSIDL_TYPESUPPORT_INTROSPECTION_CPP_PUBLIC MessageMembers_s
 {
+  /// The namespace in which the interface resides, e.g. for
+  /// example_messages/msg the namespaces generated would be
+  /// "example_messages__msg" in C and "example_message::msg" in C++.
```

Comment:
since this is the C++ API header, I think the C example can be left out.

Reply:
Agree ec4b833